addons/pos_restaurant/static/tests/tours/FloorScreen.tour.js

```diff
@@ -36,6 +36,17 @@ registry.category("web_tour.tours").add("FloorScreenTour", {
             FloorScreen.clickTable("1"),
             FloorScreen.selectedTableIs("1"),
 
+            // copy floor
+            FloorScreen.clickFloor("Main Floor"),
+            FloorScreen.clickEditButton("Copy"),
+            {
+                content: `Verify copy floor exist`,
+                trigger: 'div.floor-selector button:contains("Main Floor (copy)")',
+                isCheck: true,
+            },
+            FloorScreen.clickEditButton("Delete"),
```

Comment:
before deleting the new floor, can you add some more steps that test if the new floor contains the correct tables?